lnst/Tests/LinuxPerf.py

```diff
@@ -36,6 +36,13 @@ def run(self) -> bool:
             log_output(logging.debug, "Stderr", stderr.decode())
 
         self._res_data["filename"] = os.path.abspath(self.params.output_file)
+
+        if is_rpm_installed("kernel-debuginfo"):
+            exec_cmd(f"perf archive {self._res_data['filename']}")
```

Comment:
how quick is this command? Since we're running this test module in parallel with other measurements, i'm wondering if the addition of this command can throw off the other measurements duration?

Reply:
Don't think it takes a long time. Also, linuxperf starts before any measurement and ends after all measurements ended (unless someone changes that by adding a different measurement before linuxperf into the hierarchy).

Reply:
right, i just want to check that this doesn't throw off any of the measurement alignment that we do.